clients/lsp-xml.el

```diff
@@ -182,20 +182,33 @@ Newlines and excess whitespace are removed."
   ("xml.catalogs" lsp-xml-catalogs)
   ("xml.trace.server" lsp-xml-trace-server)))
 
-(defcustom lsp-xml-jar-file (expand-file-name
-                             (locate-user-emacs-file
-                              "org.eclipse.lsp4xml-0.3.0-uber.jar"))
+(defconst lsp-xml-jar-version "0.13.1")
+
+(defconst lsp-xml-jar-name (s-concat "org.eclipse.lemminx-" lsp-xml-jar-version "-uber.jar"))
+
+(defcustom lsp-xml-jar-file (f-join lsp-server-install-dir "xmlls" lsp-xml-jar-name)
   "Xml server jar command."
   :type 'string
   :group 'lsp-xml
   :type 'file
   :package-version '(lsp-mode . "6.1"))
 
+(defcustom lsp-xml-jar-download-url
+  (s-concat
```

Comment:
Same here, `format` is easier to see how the final url looks like

Reply:
73442c84